pkg/commands/compute/build.go

```diff
@@ -81,6 +91,12 @@ func (c *BuildCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		return err
 	}
 
+	wasmtools, err := GetWasmTools(spinner, out, c.wasmtoolsVersioner, c.Globals)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(io.Discard, "%#v\n", wasmtools)
```

Comment:
Yeah this was an old commit now you've commented on, but at the time I needed to reference `wasmtools` otherwise I couldn't compile the program to run tests but couldn't print to stdout as it would end up _breaking_ said tests 😆 so decided to just discard it.